app/views/archive-view.js

```diff
@@ -33,7 +34,8 @@ define([
     },
 
     behaviors: {
-      Navigation: {},
+      TransitionIn: {},
+      TransitionedNavigation: {},
```

Comment:
I'd still love to be able to use a single `Navigation` behavior and freely mix it with a `TransitionOut` behavior.

Here's what I'd love to be able to declare:

```
behaviors: {
    Navigation: { waitForTransition: true },
    TransitionIn: {},
    TransitionOut: {},
},
```

The flag would determine whether the Navigation itself should trigger `navigate`. If true, it would leave it to another behavior (or the view) to trigger `navigate`.